src/Tasks/SystemState.cs

```diff
@@ -192,22 +196,24 @@ public void GetObjectData(SerializationInfo info, StreamingContext context)
                     info.AddValue("fnId", frameworkName.Identifier);
                     info.AddValue("fmProf", frameworkName.Profile);
                 }
+                info.AddValue("mvid", ModuleVersionID);
             }
 
             /// <summary>
             /// Gets the last modified date.
             /// </summary>
             /// <value></value>
-            internal DateTime LastModified
+            public DateTime LastModified
```

Comment:
These were made public as part of trying to switch to json. It seemed to be serializing/deserializing only public properties in my testing console even if it had access, but I think I'm still missing a piece.